tests/ops/service/connectors/test_bigquery_connector.py

```diff
@@ -181,3 +185,40 @@ def test_retrieve_partitioned_data(
 
         assert len(results) == 1
         assert results[0]["email"] == "customer-1@example.com"
+
+    def test_retrieve_partitioned_data_with_multiple_identifying_fields(
+        self,
+        bigquery_example_test_dataset_config_with_namespace_and_partitioning_meta: DatasetConfig,
+        execution_node_with_namespace_and_partitioning_meta,
+        policy,
+        privacy_request_with_email_identity,
+        loguru_caplog,
+    ):
+        """Unit test of BigQueryQueryConfig.generate_delete specifically for a partitioned table with multiple identifying fields"""
+        dataset_config = (
+            bigquery_example_test_dataset_config_with_namespace_and_partitioning_meta
+        )
+        connector = BigQueryConnector(dataset_config.connection_config)
+
+        with loguru_caplog.at_level(logging.INFO):
+            results = connector.retrieve_data(
+                node=execution_node_with_namespace_and_partitioning_meta,
+                policy=policy,
+                privacy_request=privacy_request_with_email_identity,
+                request_task=RequestTask(),
+                input_data={
+                    "email": ["customer-1@example.com"],
+                    "custom_id": ["123"],
+                },
+            )
+            assert (
+                "SELECT address_id, created, custom_id, email, id, name FROM `silken-precinct-284918.fidesopstest.customer` WHERE (email = %(email)s OR custom_id = %(custom_id)s) AND (`created` > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1000 DAY) AND `created` <= CURRENT_TIMESTAMP())"
```

Comment:
This looks good. It does feel awkward to be reading from the _logs_ for the assertion though...?